spec/prog/vm/github_runner_spec.rb

```diff
@@ -66,6 +66,24 @@
       expect(vm.project_id).to eq(Config.github_runner_service_project_id)
     end
 
+    it "uses ch_version 46.0 without hugepages if randomly selected" do
```

Comment:
Is it true? It looks like you removed hugepage changes

Reply:
Good catch.  I rebased and updated the spec descriptions.